docs/03-extensions/external-login-handlers/providers/_category_.yml

```diff
@@ -0,0 +1,6 @@
+label: Providers
+collapsible: true
+link:
+  type: generated-index
+  description: Here's a list of our officially maintained providers
```

Comment:
**question(not-blocking):** can we easily add a paragraph with a call to action for integrators having published their own ones? So they can contact us to get it listed here.

Reply:
Added a small text about this